tests/test_data.py

```diff
@@ -41,16 +41,8 @@ def test_pg_is_local_access(postgresql, data):
     assert data.pg_is_local_access()
 
 
-def test_pg_get_db_info(data):
-    dbinfo = data.pg_get_db_info(None)
-    assert set(dbinfo) == {
-        "timestamp",
-        "no_xact",
-        "total_size",
-        "max_length",
-        "tps",
-        "size_ev",
-    }
+def pg_get_server_informations(data):
```

Comment:
Don't really know how to go about this given : 

1. the amount of data to test (not a real excuse, I am just being lazy)
2. the variance between (supported) versions
3. the mess it's gona be to create meaningful tests for each and every info (when it's even possible) AND keep them functional across PostgreSQL version (some are optimize specific).

Reply:
Maybe we can keep the same simple sanity check as before (i.e. only make sure that we have expected fields)?

Reply:
I think that's what I did.

We used to return all the fields to the caller and do the object creation there. 
I thought it was cleaner to return the object directly. So now if we're missing filed in the select
the object creation will fail.

Some fields might be `None` because across versions some info are missing.

Reply:
Sorry, I think I misunderstood your question given the diff context. Can you reformulate?

Reply:
dealt with irl